concent_api/signing_service_integration_test_middleman.py

```diff
@@ -75,6 +90,84 @@ def test_case_send_golem_message_frame_with_correct_transaction_signing_request(
     assert_condition(type(response.payload), SignedTransaction, f'Deserialized response payload type is {type(response.payload)} instead of SignedTransaction.')
 
 
+@count_fails
+def test_case_2_send_golem_message_frame_with_incorrect_transaction_signing_request_do_not_kill_communication(config: ConfigParser, **kwargs: Any) -> None:
```

Comment:
I don't understand the "_do_not_kill_communication" part of the TC name - could you please explain?

Reply:
hm... in such case, maybe `test_case_2_send_golem_message_frame_with_incorrect_then_correct_transaction_signing_request` would be more precise? I am not sure here :)

Reply:
:+1: done